.gitignore

```diff
@@ -74,6 +75,19 @@ series
 cscope.*
 ncscope.*
 
+# Gator daemon Files
```

Comment:
Those ones I would put them at the bottom of the gitignore file, along with the other Broadcom ignores, to still not diverge too much from QC.